hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/NotificationMessage.java

```diff
@@ -65,7 +65,7 @@ private void decorate(final String styleName, final String caption, final String
         setCaption(caption);
         setDescription(description);
         setStyleName(styleName);
-        setHtmlContentAllowed(true);
+        setHtmlContentAllowed(false);
```

Comment:
Have you verified non of the components are using HTML content in notifications not to brake them after it is disallowed?

Reply:
Newest change is pushed regarding < br > in notification, so now every notification is checked and works fine.